.gitignore

```diff
@@ -65,6 +65,9 @@
 *.vcxproj.user
 enc_temp_folder/*
 CMakeSettings.json
+*PVS-Studio*
+*pvs-studio*
+*PVS*
```

Comment:
Does it work if you keep only ``*pvs*``?

Reply:
id imagine so.. :*

Reply:
You mean, it doesn't ignore all files?

Reply:
no it does, i just didnt think about it that way

Reply:
*PVS* may be a too generic pattern, especially in RPCS3 where we use abbreviations alot, is it possible for it to be more specific?

Reply:
just double checked, it needs to check for all files including `PVS-Studio`. it creates lock files as it traverses the codebase, which should be ignored

Reply:
PVS/* allows you to create a PVS workspace inside of the project and have that ignored too.